app/javascript/flavours/glitch/components/avatar.tsx

```diff
@@ -0,0 +1,54 @@
+import * as React from 'react';
+import classNames from 'classnames';
+import { autoPlayGif } from '../initial_state';
+import { useHovering } from '../../../hooks/useHovering';
+import type { Account } from '../../../types/resources';
```

Comment:
I really don't like doing imports like this, but changing tsconfig.json to allow 'flavours/glitch/...' type imports feels out of scope for this PR and also makes some of the ts-expect-error comments obsolete.